api/src/main/java/net/starype/quiz/api/game/answer/DoubleValidityEvaluator.java

```diff
@@ -0,0 +1,23 @@
+package net.starype.quiz.api.game.answer;
+
+public class DoubleValidityEvaluator implements ValidityEvaluator {
+    private DoubleValidityEvaluator() {}
+
+    private static DoubleValidityEvaluator instance = null;
+
+    public static DoubleValidityEvaluator getInstance()
+    {
+        if(instance == null) {
+            instance = new DoubleValidityEvaluator();
+        }
+        return instance;
+    }
+
+    @Override
+    public boolean isValid(Answer answer) {
+        return answer
+                .getAnswerText()
+                .strip()
+                .matches("^[+-]?[0-9]*[,.]?[0-9]*$");
```

Comment:
A regex isn't the best way of verifying this. For instance, `2349238472937482374982374897238472389478927289342983` would match but is not a valid double (it is way too big!). A try catch may be better